mods/doors/init.lua

```diff
@@ -734,6 +734,16 @@ function doors.register_fencegate(name, def)
 		},
 	}
 
+
+	if type(def.texture) == "string" then
+		fence.tiles[1] = {name = def.texture, backface_culling = true}
+	elseif def.texture.backface_culling == nil then
```

Comment:
`elseif def.texture.backface_culling != true then`
To override if it is set to 'false'?